README.md

```diff
@@ -1,220 +1,130 @@
-We do our best to identify and fix problems, and implement missing
-features.
+This implementation has been in production use on the Bitcoin mainnet since early 2018, with the launch of the [Blockstream Store][blockstream-store-blog].
+We recommend getting started by experimenting on `testnet`, but the implementation is considered stable and can be safely used on mainnet.
 
-Any help testing the implementation, reporting bugs, or helping with
-outstanding issues is very welcome.
-Don't hesitate to reach out to us on IRC at
-[#lightning-dev @ freenode.net][irc1], [#c-lightning @
-freenode.net][irc2], or on the implementation-specific mailing list
-[c-lightning@lists.ozlabs.org][ml1], or on the Lightning Network-wide
-mailing list [lightning-dev@lists.linuxfoundation.org][ml2].
-
-[irc1]: http://webchat.freenode.net/?channels=%23lightning-dev
-[irc2]: http://webchat.freenode.net/?channels=%23c-lightning
-[ml1]: https://lists.ozlabs.org/listinfo/c-lightning
-[ml2]: https://lists.linuxfoundation.org/mailman/listinfo/lightning-dev
+Any help testing the implementation, reporting bugs, or helping with outstanding issues is very welcome.
+Don't hesitate to reach out to us on IRC at [#lightning-dev @ freenode.net][irc1], [#c-lightning @ freenode.net][irc2], or on the implementation-specific mailing list [c-lightning@lists.ozlabs.org][ml1], or on the Lightning Network-wide mailing list [lightning-dev@lists.linuxfoundation.org][ml2].
 
 ## Getting Started
 
-c-lightning currently only works on Linux (and possibly Mac OS with some
-tweaking), and requires a locally (or remotely) running `bitcoind` (version 0.15 or
-above) that is fully caught up with the network you're testing on.
-Pruning (prune=n option in bitcoin.conf) is not currently supported.
+c-lightning only works on Linux and Mac OS, and requires a locally (or remotely) running `bitcoind` (version 0.16 or above) that is fully caught up with the network you're testing on.
+Pruning (`prune=n` option in `bitcoin.conf`) is partially supported, see [here](#pruning) for more details.
 
 ### Installation
 
-Please refer to the [installation documentation](doc/INSTALL.md) for
-detailed instructions.
-For the impatient here's the gist of it for Ubuntu and Debian:
+There are 4 supported installation options:
 
-    sudo apt-get update
-    sudo apt-get install -y \
-      autoconf automake build-essential git libtool libgmp-dev \
-      libsqlite3-dev python python3 net-tools zlib1g-dev libsodium-dev \
-      python3-mako
-    git clone https://github.com/ElementsProject/lightning.git
-    cd lightning
-    ./configure
-    make
+ - Installation from the [Ubuntu PPA][ppa]
+ - Installation of a pre-compiled binary from the [release page][releases] on Github
+ - Using one of the [provided docker images][dockerhub] on the Docker Hub
+ - Compiling the source code yourself (suggested mainly for developers or if you need one of the still [unreleased features][changelog-unreleased])
```

Comment:
do we have those lines 
```
sudo apt-get update
    sudo apt-get install -y \
      autoconf automake build-essential git libtool libgmp-dev \
      libsqlite3-dev python python3 net-tools zlib1g-dev libsodium-dev \
      python3-mako
    git clone https://github.com/ElementsProject/lightning.git
    cd lightning
    ./configure
    make
``` 
somewhere else in the `doc` folder? if yes we should link to them (instead of the changelog) otherwise we should create such a file. This again was and is extremely helpful for people and something the look at. I might even suggest to leave this inside the readme

Reply:
doc/INSTALL.md, yes!  A link would help here, for sure.